### ISSUE TITLE
Graticule does not work in 2D

### ISSUE THREAD

**Issue by jepessen** [gwaldron/osgearth#1437]
Graticule at the moment works only in geocentric mode, not in projected mode.
I'm seeing that it's a TODO in the documentation of `osgEarth::Util::Graticule` class, but I'd like to know if anyone is working on it, or what I can do in order to implement this feature, because we need it. What's the reason for which is working only in geocentric mode?

**Comment by gwaldron** [comment]
No one's working on it afaik. Feel free to dive in and give it a whirl!

**Comment by jepessen** [comment]
I'm trying. I've forked the project. It builds by using Visual Studio and vcpkg, but when I start, for example, osgearth_tocd.exe I obtain the following error:

```{text}
The procedure entry point ?White@color@osgEarth@@2U12@A could not be located in the dynamic library build/path/osgdb_osgearth_engine_rexd.dll
```
I've checked a bit, and it seems that the symbol is defined in `osgearth\src\osgEarth\Color`, and osgEarthd.dll is in the same working folder, so I don't know why I'm obtaining this error.

Any suggestion?
